addon/components/search-result/template.hbs

```diff
@@ -0,0 +1,124 @@
+{{! Template similar to Ember-Preprints Layout, but using Ember-SHARE properties}}
+<div class="search-result p-sm">
+    <div class="row">
+        <div class="col-xs-12">
+            {{!Title}}
+            <h4>
+                {{#if (and detailRoute osfID)}}
+                    {{#link-to detailRoute osfID invokeAction=(action 'click' 'link' 'Discover - Result Title' osfID)}}{{fix-special-char result.title}}{{/link-to}}
+                {{else if (and detailRoute (eq hostAppName 'Retraction Watch'))}}
+                    {{#link-to detailRoute result.id}}{{fix-special-char result.title}}{{/link-to}}
+                {{else}}
+                    {{#if hyperlink}}
+                        <a href='{{hyperlink}}' onclick={{action 'click' 'link' (concat 'Discover - Result Title: ' hyperlink) result.id}}>{{fix-special-char result.title}}</a>
```

Comment:
`Discover - Result Title:` is being sent to Google Analytics and does not need translation.